modules/affinity/pom.xml

```diff
@@ -33,6 +33,12 @@
     <version>3.0.0-SNAPSHOT</version>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-raft</artifactId>
+            <version>${project.version}</version>
```

Comment:
Here and below. Version should be removed completely. See https://issues.apache.org/jira/browse/IGNITE-14630

Reply:
But the issue is Open now.
I think all will be fixed in it.

Reply:
Well, cause we already know that it should be removed, I believe it's better to fix it within every new PR.